src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,127 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Ui {
+  Scanner io;
+
+  /**
+   * Initialise the Ui object with a Scanner object.
+   */
+  Ui() {
+    this.io = new Scanner(System.in);
+  }
+
+  /**
+   * Asks the user for input
+   * @return String representation of said input.
+   */
+  public String getCommand() {
+    return io.nextLine();
+  }
+
+  /**
+   * Prints the Greeting.
+   */
+  public void greeting() {
+    System.out.println("----------------------------");
+    System.out.println("Hello I'm your task manager!");
+    System.out.println("----------------------------\n");
+  }
+
+  /**
+   * Prints the Initial Prompt.
+   */
+  public void initPrompt() {
+    System.out.println("\n----------------------------");
+    System.out.println("What tasks do you have dude?");
+    System.out.println("----------------------------");
+  }
+
+  /**
+   * Prints a small line
+   */
+  public void printSmallLine() {
```

Comment:
Nice way to modularise the function to execute `System.out.println("    -----");`